pkg/haconfig/interfaces.go

```diff
@@ -0,0 +1,17 @@
+package haconfig
```

Comment:
I think ipfailover is strictly better than haconfig, but if you have a different suggestion let me know.  Other options are vipmanager or vipfailover - I didn't brainstorm extensively.

----- Original Message -----
> > @@ -0,0 +1,17 @@
> > +package haconfig
> 
> Will do and change it across the board.
> 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/openshift/origin/pull/1807/files#r28801481